components/geometry-merger/examples/buffer/index.html

```diff
@@ -4,17 +4,27 @@
     <meta charset="utf-8">
     <title>A-Frame Geometry Merger Component - Buffer</title>
     <meta name="description" content="Merging buffer geometry with Geometry Merger component."></meta>
-    <script src="https://aframe.io/releases/0.8.2/aframe.js"></script>
+    <script src="https://aframe.io/releases/1.4.1/aframe.js"></script>
+    <script>
+      // This is a patch for an A-Frame bug not yet fixed in the latest published version:
+      // https://github.com/aframevr/aframe/pull/5250
+      const materialPrototype = AFRAME.components.material.Component.prototype
+      materialPrototype.updateMaterialOriginal = materialPrototype.updateMaterial;
+      materialPrototype.updateMaterial = function (oldData) {
+        this.updateMaterialOriginal(oldData)
+        this.material.vertexColors = (this.data.vertexColors === 'vertex')
+      }
+    </script>
     <script src="../../dist/aframe-geometry-merger-component.min.js"></script>
-    <script src="vertex-colors-buffer.js"></script>
+    <script src="../components/vertex-colors-buffer.js"></script>
```

Comment:
vertex-colors-buffer.js now used by both examples so moved to a more suitable location